purchase_all_shipments/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+======================
+Purchase All Shipments
+======================
+
+With the core "purchase" module, in a purchase order a button "In Shipments"
+lets the user see the picking that was generated by the order itself.
+
+With this module, that button is replaced by an "All Shipments" button that
+shows the original picking, and all others that are grouped with it. This
+should include pickings associated to the procurements that generated the
```

Comment:
>  the procurements 

Use "The procurement group" instead since there is only one, and is a procurement group not deprecated model procurement: could be confusion